apps/testnorge-skd/skd-core/src/main/java/no/nav/registre/skd/service/SyntetiseringService.java

```diff
@@ -42,6 +42,9 @@
 @DependencyOn("testnorge-hodejegeren")
 public class SyntetiseringService {
 
+    static final String AVSPILLERGRUPPE = " i avspillergruppe ";
+    static final String SKDMELDING_ID_STRING = ". Skdmeldinger som er lagret i TPSF, men som ikke ble sendt til TPS har følgende id-er i TPSF: ";
+    static final String SKDMELDINGER_TPSF_STRING = " - Skdmeldinger som ble lagret i TPSF: ";
```

Comment:
Disse tre kan du egentlig legge sammen i én konstant, og ha %s og %d for å fylle den med verdier der den logges ved å bruke String.format(). (les mer om den her: https://dzone.com/articles/java-string-format-examples )

Så det blir noe sånt som:
`private static final FEILMELDING = "Feil på endringskode %s i avspillergruppe %d. Skdmeldinger som er lagret i TPSF, men som ikke ble sendt til TPS har følgende id-er i TPSF: %s - Skdmeldinger som ble lagret i TPSF: %s`

Så skriver du der feilmeldingene kastes noe sånt som:
`httpStatus = loggExceptionOgLeggTilFeiletEndringskode(e, "ManglendeInfoITPSException: " + String.format(FEILMELDING, endringskode.getEndringskode(), genereringsOrdreRequest.getAvspillergruppeId(), lagGrupperAvIder(idsLagretITpsfMenIkkeTps), lagGrupperAvIder(ids)), endringskode.getEndringskode());`

Reply:
Genialt 🤩  Takk, hadde nok stuka litt med den lange der 😅